Resources/fsh-generated/resources/StructureDefinition-ISiKKontaktGesundheitseinrichtung.json

```diff
@@ -299,7 +299,7 @@
         "path": "Encounter.type",
         "sliceName": "Kontaktebene",
         "short": "Kontaktebene",
-        "comment": "Das Konzept der 'Kontaktebene' stammt aus dem Fallmodell der Medizininformatik-Initiative, \n  das bei Encountern zwischen 'Einrichtungskontakten', 'Fachabteilungskontakten' und 'Versorgungsstellenkontakten' unterscheidet.\n  Im Kontext dieses Moduls werden lediglich Encounter der Ebene 'Fachabteilungskontakt' abgebildet.\n  **Begründung Pflichtfeld:** Die Abteilungsebene muss aus Kompatibilitätsgründen angegeben werden.",
+        "comment": "  **Begründung Pflichtfeld:** Die Abteilungsebene muss aus Kompatibilitätsgründen angegeben werden.\n  \n    **Hinweis Bei Abbildung von Versorgungsstellenkontakten:**\n  Es ist ein üblicher Fall, dass die Dauer eines Versorgungsstellenkontaktes in der Versorgung die eines Abteilungskontaktes übersteigt. Ein Beispiel hierfür: Ein Patient bleibt im Bett (Versorgungsstellenkontakt), aber ein Fachabteilungswechsel geschieht, da die Diagnose über eine Fachabteilung (Onkologie) läuft, dann aber der Wechsel zur Fachabteilung Chirurgie (neuer Abteilungskontakt) notwendig wird.\n  Für einen solchen Fall gilt auf Ebene der FHIR-Instanzen (z.B. entgegen des tatsächliche Aufenthaltes im gleichen Bett): Im Falle eines Fachabteilungswechsels legt ein System einen neuen Abteilungskontakt an. Bestehende Versorgungsstellenkontakt SOLLEN NICHT in ihrer Relation (.partOf) zum Abteilungskontakt modifiziert werden. Hingegen SOLL das System einen oder mehrere Versorgungsstellenkontakte erzeugen und mit dem neu angelegten Abteilungskontakt in Verbindung setzen.\n\n  Hintergrund: Das Konzept der 'Kontaktebene' stammt aus dem Fallmodell der Medizininformatik-Initiative, \n    das bei Encountern zwischen 'Einrichtungskontakten', 'Fachabteilungskontakten' und 'Versorgungsstellenkontakten' unterscheidet.\n    Im Kontext dieses Moduls werden lediglich Encounter der Ebene 'Fachabteilungskontakt' abgebildet.\n  \n  ",
```

Comment:
Würde den Teil ab 'Hintergrund: ' voranstellen und dann erst auf die Abbildung von Versorgungsstellenkontakten eingehen.

Reply:
suggestion. ich würde es so lasse, da das Thema Fall-Konzepte schon in der Motivation genannt wird (inklusive Referenz).